contrib/utilities/indent

```diff
@@ -15,12 +15,14 @@
 ## ---------------------------------------------------------------------
 
 #
-# This script does the same thing as contrib/utilities/indent but only
+# This script does the same thing as contrib/utilities/indent-all but only
 # reformats files which have changed (or have been added but neither
 # staged/commited) since the last merge commit to the master branch.
 #
 # The script needs to be executed as
 #   ./contrib/utilities/indent
+# or
+#   ./contrib/utilities/indent clang_format_command
```

Comment:
I would prefer this to be controlled by an environment variable, let's say `DEAL_II_CLANG_FORMAT`.
Problem is that you cannot supply this parameter when running `make indent`, or `make indent_all`, further supplying the indent binary as first parameter to a script that is called `indent` is just too confusing.